### DISCUSSION TITLE
não consigo baixar video

### DISCUSSION THREAD

**Discussion by adautoar** [aclap-dev/video-downloadhelper#595] [Help and Q&A]
### Your question

Olá comprei um curso e estava baixando os videos. Nessa nova versão, tento baixar os videos e ocorre erro. gostaria de ajuda para solucionar, obrigado

### Addon & CoApp version and details

{
  "version": "9.0.1.18",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://app.nutror.com/curso/8639b1d044289ed4888c09b83f9024834cd99be3/aula/5909716",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 5 times\r\n[in#0 @ 000001f37d228e40] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJmYzkzMmQ3YS00NjhkLTQ0NTctOTUyNy0zYjg3MjhmODkwNzciLCJzZXNzaW9uS2V5IjoieXN4cTVmaG0iLCJwcm9kdWNlcktleSI6IjIyNjMxODQ4IiwicHJvZHVjZXJFbWFpbCI6InN1cG9ydGVAY3Vyc29zZHouY29tIiwiY29uc3VtZXJFbWFpbCI6InJvZHJpZ29oZW5yaXF1ZTA4QHlhaG9vLmNvbS5iciIsImltcHJpbnQiOnRydWUsInN0b3JhZ2UiOiJyMiIsInBsIjoiNzIwcF9wbGF5bGlzdC5tM3U4IiwiaWF0IjoxNzE2NTE4OTYzLCJleHAiOjE3MTY2OTE3NjN9.vdbw5OG4GK4_sSu86ASokSngrr7qrfzSUNth22_a1Jw.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_8201370254220533",
    "error": "coapp_failure",
    "id": "coapp_failure_ac1181a3-08a8-40cb-bb74-864b9d0fd404"
  }
}
version: 9.0.1.18
target: microsoft
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"1106ba4f2e144029"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1
}

**Comment by paulrouget** [comment]
Triage: ff-failure

> non standard extension and non standard mime type
> Invalid data

**Comment by adautoar** [comment]
Olá, antes eu conseguia baixar os vídeos, como proceder para download agora?obrigado
    Em sexta-feira, 24 de maio de 2024 às 00:06:03 BRT, Paul ***@***.***> escreveu:  
 
 


Triage: ff-failure


non standard extension and non standard mime type
Invalid data
